### PR TITLE
new bootnodes

### PR DESCRIPTION
**Pull request by ETeissonniere**

### CODE REVIEW

File: node/res/eden.json [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
-	"/dns4/node-6907887383360909312-0.p2p.onfinality.io/tcp/25247/p2p/12D3KooWBDuW17r8nffVwBLB6aiCioTbDYJPnYaxBF84UJpuPfKB"
+	"/dns4/node-6973269498897903616-0.p2p.onfinality.io/tcp/23024/p2p/12D3KooWRCdwNbBYyFATQJqnz4v2LxwNgPCW8924CeQxLkUxAhBq",
+	"/dns4/node-6973269770037710848-0.p2p.onfinality.io/tcp/21138/p2p/12D3KooWDjqypbSf9eutZ1K1y4VpB22sWJYaMdTAd5dxR1jzFStS",
+	"/dns4/node-6973269893510606848-0.p2p.onfinality.io/tcp/27619/p2p/12D3KooWGU6SbCJQQW5XfwLzxe7KuWMAP2fEcA2xwQsrLE3oLf3G",
+	"/dns4/node-6973270084778283008-0.p2p.onfinality.io/tcp/28366/p2p/12D3KooWL3sZ1d3W7vQNx9hC8LWo9msvo8GThLH2wyUsmYnXHyD1",
+	"/dns4/node-6973270204732157952-0.p2p.onfinality.io/tcp/21169/p2p/12D3KooWFBURotJ2YK14xDyDNytkjj5KgtatRLaJd1wEEox2h2cK",
+	"/dns4/node-6975196169949773824-0.p2p.onfinality.io/tcp/22621/ws/p2p/12D3KooWMv1jzb45EMGZVGjqgoAjmqPrsYchHCQ13McFKL1r75b5",
+	"/dns4/node-6975196240558477312-0.p2p.onfinality.io/tcp/25394/ws/p2p/12D3KooW9tfcevBR9DjD1JznXY7TenxVoDvtuVKndjgWNizjGUCs",
+	"/dns4/node-6975196308969979904-0.p2p.onfinality.io/tcp/18424/ws/p2p/12D3KooW9zvqhwkgoPihdnHAeGY6SN3xxxXY8HTmwUxpAYdANS4i"
```
Comment (simonsso) [inline node/res/eden.json:13]:
```suggestion
	"/dns4/node-6975196308969979904-0.p2p.onfinality.io/tcp/18424/p2p/12D3KooW9zvqhwkgoPihdnHAeGY6SN3xxxXY8HTmwUxpAYdANS4i"
```

File: node/res/eden.json [hunk 2]
```diff
@@ -3,11 +3,14 @@
...
-	"/dns4/node-6907887240700223488-0.p2p.onfinality.io/tcp/21455/p2p/12D3KooWK4ZT2BEVDu83LFDTzdQmHuHPR9Zd9YPitDBAh3ZN5NU1",
-	"/dns4/node-6907887383360909312-0.p2p.onfinality.io/tcp/25247/p2p/12D3KooWBDuW17r8nffVwBLB6aiCioTbDYJPnYaxBF84UJpuPfKB"
+	"/dns4/node-6973269498897903616-0.p2p.onfinality.io/tcp/23024/p2p/12D3KooWRCdwNbBYyFATQJqnz4v2LxwNgPCW8924CeQxLkUxAhBq",
+	"/dns4/node-6973269770037710848-0.p2p.onfinality.io/tcp/21138/p2p/12D3KooWDjqypbSf9eutZ1K1y4VpB22sWJYaMdTAd5dxR1jzFStS",
+	"/dns4/node-6973269893510606848-0.p2p.onfinality.io/tcp/27619/p2p/12D3KooWGU6SbCJQQW5XfwLzxe7KuWMAP2fEcA2xwQsrLE3oLf3G",
+	"/dns4/node-6973270084778283008-0.p2p.onfinality.io/tcp/28366/p2p/12D3KooWL3sZ1d3W7vQNx9hC8LWo9msvo8GThLH2wyUsmYnXHyD1",
+	"/dns4/node-6973270204732157952-0.p2p.onfinality.io/tcp/21169/p2p/12D3KooWFBURotJ2YK14xDyDNytkjj5KgtatRLaJd1wEEox2h2cK",
+	"/dns4/node-6975196169949773824-0.p2p.onfinality.io/tcp/22621/ws/p2p/12D3KooWMv1jzb45EMGZVGjqgoAjmqPrsYchHCQ13McFKL1r75b5",
+	"/dns4/node-6975196240558477312-0.p2p.onfinality.io/tcp/25394/ws/p2p/12D3KooW9tfcevBR9DjD1JznXY7TenxVoDvtuVKndjgWNizjGUCs",
```
Comment (simonsso) [inline node/res/eden.json:12]:
```suggestion
	"/dns4/node-6975196240558477312-0.p2p.onfinality.io/tcp/25394/p2p/12D3KooW9tfcevBR9DjD1JznXY7TenxVoDvtuVKndjgWNizjGUCs",
```

File: node/res/eden.json [hunk 3]
```diff
@@ -3,11 +3,14 @@
...
-	"/dns4/node-6907887090221178880-0.p2p.onfinality.io/tcp/14424/p2p/12D3KooWK52WwaMXpBDMthfM9LM2HwrEwLKoc57kgW3ZdvjqB8om",
-	"/dns4/node-6907887240700223488-0.p2p.onfinality.io/tcp/21455/p2p/12D3KooWK4ZT2BEVDu83LFDTzdQmHuHPR9Zd9YPitDBAh3ZN5NU1",
-	"/dns4/node-6907887383360909312-0.p2p.onfinality.io/tcp/25247/p2p/12D3KooWBDuW17r8nffVwBLB6aiCioTbDYJPnYaxBF84UJpuPfKB"
+	"/dns4/node-6973269498897903616-0.p2p.onfinality.io/tcp/23024/p2p/12D3KooWRCdwNbBYyFATQJqnz4v2LxwNgPCW8924CeQxLkUxAhBq",
+	"/dns4/node-6973269770037710848-0.p2p.onfinality.io/tcp/21138/p2p/12D3KooWDjqypbSf9eutZ1K1y4VpB22sWJYaMdTAd5dxR1jzFStS",
+	"/dns4/node-6973269893510606848-0.p2p.onfinality.io/tcp/27619/p2p/12D3KooWGU6SbCJQQW5XfwLzxe7KuWMAP2fEcA2xwQsrLE3oLf3G",
+	"/dns4/node-6973270084778283008-0.p2p.onfinality.io/tcp/28366/p2p/12D3KooWL3sZ1d3W7vQNx9hC8LWo9msvo8GThLH2wyUsmYnXHyD1",
+	"/dns4/node-6973270204732157952-0.p2p.onfinality.io/tcp/21169/p2p/12D3KooWFBURotJ2YK14xDyDNytkjj5KgtatRLaJd1wEEox2h2cK",
+	"/dns4/node-6975196169949773824-0.p2p.onfinality.io/tcp/22621/ws/p2p/12D3KooWMv1jzb45EMGZVGjqgoAjmqPrsYchHCQ13McFKL1r75b5",
```
Comment (simonsso) [inline node/res/eden.json:11]:
```suggestion
	"/dns4/node-6975196169949773824-0.p2p.onfinality.io/tcp/22621/p2p/12D3KooWMv1jzb45EMGZVGjqgoAjmqPrsYchHCQ13McFKL1r75b5",
```

**Review by simonsso — COMMENTED** [review]
The new nodes looks ok, peers gets connected with only them.

`ws/` was not part of the old bootnodes and I don't think we need it in the new ones.

**Review by simonsso — DISMISSED** [review]

File: node/res/eden.json [hunk 4]
```diff
@@ -3,11 +3,11 @@
...
-	"/dns4/node-6907886965943775232-0.p2p.onfinality.io/tcp/16773/p2p/12D3KooWS8y5U8YhemFZ9mxXAX1RDxq3WifBGWcG9YTBDNpc9BUG",
-	"/dns4/node-6907887090221178880-0.p2p.onfinality.io/tcp/14424/p2p/12D3KooWK52WwaMXpBDMthfM9LM2HwrEwLKoc57kgW3ZdvjqB8om",
-	"/dns4/node-6907887240700223488-0.p2p.onfinality.io/tcp/21455/p2p/12D3KooWK4ZT2BEVDu83LFDTzdQmHuHPR9Zd9YPitDBAh3ZN5NU1",
-	"/dns4/node-6907887383360909312-0.p2p.onfinality.io/tcp/25247/p2p/12D3KooWBDuW17r8nffVwBLB6aiCioTbDYJPnYaxBF84UJpuPfKB"
+	"/dns4/node-6973269498897903616-0.p2p.onfinality.io/tcp/23024/p2p/12D3KooWRCdwNbBYyFATQJqnz4v2LxwNgPCW8924CeQxLkUxAhBq",
+	"/dns4/node-6973269770037710848-0.p2p.onfinality.io/tcp/21138/p2p/12D3KooWDjqypbSf9eutZ1K1y4VpB22sWJYaMdTAd5dxR1jzFStS",
+	"/dns4/node-6973269893510606848-0.p2p.onfinality.io/tcp/27619/p2p/12D3KooWGU6SbCJQQW5XfwLzxe7KuWMAP2fEcA2xwQsrLE3oLf3G",
+	"/dns4/node-6973270084778283008-0.p2p.onfinality.io/tcp/28366/p2p/12D3KooWL3sZ1d3W7vQNx9hC8LWo9msvo8GThLH2wyUsmYnXHyD1",
+	"/dns4/node-6973270204732157952-0.p2p.onfinality.io/tcp/21169/p2p/12D3KooWFBURotJ2YK14xDyDNytkjj5KgtatRLaJd1wEEox2h2cK"
```
Comment (simonsso) [inline node/res/eden.json]: There are no peers in these nodes

File: node/res/eden.json (hunk 3 above)
Comment (simonsso) [inline node/res/eden.json:11]: The last three lines are active and works

File: node/res/eden.json (hunk 4 above)
Comment (ETeissonniere) [reply node/res/eden.json]: they should be online now :)

Comment (simonsso) [reply node/res/eden.json]: Ok I will check

Comment (simonsso) [reply node/res/eden.json]: Yes they are all alive!

**Review by simonsso — APPROVED** [review]

**Pull request merged**